themes/scss/map/cartodb-map-light.scss

```diff
@@ -309,31 +309,31 @@ only screen and (                min-resolution: 2dppx) {
     background-position:-94px 10px!important;
   }
   div.cartodb-header div.social a.facebook {
-    background:url('../img/other@2x.png') no-repeat 0 0!Important;
-    background-size: 113px 34px!Important;
+    background:url('../img/other@2x.png') no-repeat 0 0!important;
+    background-size: 113px 34px!important;
   }
   div.cartodb-header div.social a.twitter {
-    background:url('../img/other@2x.png') no-repeat -26px 0!Important;
-    background-size: 113px 34px!Important;
+    background:url('../img/other@2x.png') no-repeat -26px 0!important;
+    background-size: 113px 34px!important;
   }
   div.cartodb-searchbox span.loader {
-    background: url('../img/loader@2x.gif') no-repeat center center white!Important;
-    background-size: 16px 16px!Important;
+    background: url('../img/loader@2x.gif') no-repeat center center white!important;
+    background-size: 16px 16px!important;
   }
   div.cartodb-mobile .aside div.cartodb-searchbox span.loader {
-    background: url('../img/dark_loader@2x.gif') no-repeat center center #292929!Important;
-    background-size: 16px 16px!Important;
+    background: url('../img/dark_loader@2x.gif') no-repeat center center #292929!important;
```

Comment:
! should be preceeded by a space, and should not be followed by a space<br>!important should not be used